client/app/domain/general-informations/domain-general-informations.controller.js

```diff
@@ -467,6 +504,9 @@ angular.module('App').controller(
       }
     }
 
+    updateOwner() {
+      window.open(this.updateOwnerUrl.target, '_self');
```

Comment:
Is it possible to have the user clock on the link instead ? If not, please use `$window` instead of `window`